bootconfig/README.md

```diff
@@ -0,0 +1,53 @@
+# Bootconfig API
+
+The bootconfig service is responsible for the setting of bootstrap configuration for the device after initial Bootz bootstrap.
+This message may include both the base device configuration but also the new gNSI artifacts.
+
+While this is service which can be used out of band it also allows for the development of an industry standard CLI for recovery
+of devices.
+
+## RPC use case
+
+* From remote rpc client call `gnoi.Bootconfig.Set` using bootconfig message containing elements to override.
+
+## CLI use cases
+
+The CLI should be bundled with the router and exist in the default path.
```

Comment:
Is the cli expected to be called from the vendor cli, or a linux shell?

Reply:
shell